test/elasticsearch-utils.test.js

```diff
@@ -0,0 +1,54 @@
+const esUtils = require('../lib/elasticsearch/utils')
+
+describe('ES utils', () => {
+  describe('escapeQuery', function () {
+    it('should escape specials', function () {
+      expect(esUtils.escapeQuery('? ^ * + (')).to.equal('\\? \\^ \\* \\+ \\(')
+    })
+
+    it('should escape unrecognized field indicators', function () {
+      expect(esUtils.escapeQuery('fladeedle:gorf')).to.equal('fladeedle\\:gorf')
+    })
+
+    it('should not escape recognized field indicators', function () {
+      expect(esUtils.escapeQuery('title:gorf')).to.equal('title:gorf')
+    })
+
+    it('should escape a single forward slash', function () {
+      expect(esUtils.escapeQuery('/')).to.equal('\\/')
+    })
+
+    it('should escape floating colon', function () {
+      // Make sure colons floating in whitespace are escaped:
+      expect(esUtils.escapeQuery('Arkheologii︠a︡ Omska : illi︠u︡strirovannai︠a︡ ėnt︠s︡iklopedii︠a︡')).to.equal('Arkheologii︠a︡ Omska \\: illi︠u︡strirovannai︠a︡ ėnt︠s︡iklopedii︠a︡')
+    })
+
+    it('should escape colons in hyphenated phrases', function () {
+      expect(esUtils.escapeQuery('Arkheologii︠a︡ Omska : illi︠u︡strirovannai︠a︡ ėnt︠s︡iklopedii︠a︡ / Avtor-sostavitelʹ: B.A. Konikov.')).to.equal('Arkheologii︠a︡ Omska \\: illi︠u︡strirovannai︠a︡ ėnt︠s︡iklopedii︠a︡ \\/ Avtor\\-sostavitelʹ\\: B.A. Konikov.')
+    })
+  })
+
+  describe('prefixMatch', () => {
+    it('builds a prefix clause', () => {
```

Comment:
i think these could use a test where there is no boost

Reply:
Added